examples/experimental/export_dummy_example.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    # - ["trulens.bindings"] - Attributes specific to TruLens spans. Call spans include
</pre><p>what attributes are available here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>